packages/web-components/fast-components/src/number-field/index.ts

```diff
@@ -58,10 +58,14 @@ export const fastNumberField = NumberField.compose<NumberFieldOptions>({
         delegatesFocus: true,
     },
     stepDownGlyph: `
-        <span class="step-down-glyph" part="step-down-glyph"></span>
+        <svg width="10" height="9" viewBox="0 0 10 9" xmlns="http://www.w3.org/2000/svg">
```

Comment:
There are a lot of off-pixel numbers here. I've updated the icons in Figma. Please get the updates and optimize them again. They should be 12x12 containers.

Reply:
updated